ftw_tools/download/download_img.py

```diff
@@ -235,7 +235,7 @@ def query_stac(
     return least_cloudy_item.get_self_href()
 
 
-def create_input(win_a, win_b, out, overwrite, stac_host, bbox=None):
+def create_input(win_a, out, overwrite, stac_host, win_b=None, bbox=None):
```

Comment:
This updates the download functionality to allow for only downloading a single image instead of requiring multiple images (winA AND winB)